src/sentry/static/sentry/app/components/group/releaseChart.jsx

```diff
@@ -25,20 +25,27 @@ const GroupReleaseChart = createReactClass({
     title: PropTypes.string,
   },
 
-  getInitialState(props) {
-    if (!defined(props)) props = this.props;
+  getInitialState() {
+    return this.getNextState(this.props);
+  },
+
+  componentWillReceiveProps(nextProps) {
+    this.setState(this.getNextState(nextProps));
+  },
+
+  getNextState(props) {
     let releaseStats = props.releaseStats;
     let releasePoints = {};
-    if (defined(releaseStats)) {
-      releaseStats[this.props.statsPeriod].forEach(point => {
+    if (releaseStats) {
```

Comment:
what were the defined falsy values that were causing problems here?

Reply:
There wasn't a problem here, just trying to simplify it